### ISSUE TITLE
Improve message when executable cannot be found

### ISSUE THREAD

**Issue by kaste** [SublimeLinter/SublimeLinter#1418]
Currently we output something like `SublimeLinter: #2 linter.py:573: WARNING: lua cannot locate 'luac'` if the executable cannot be found using a global which. 

```
SublimeLinter: WARNING: (lua) Linter non-functional: Cannot find a binary 'luac' in PATH.
```

Maybe we should then print PATH as well. Or ask "Did you install 'luac' globally?"

As reported here https://github.com/SublimeLinter/SublimeLinter-lua/issues/7

**Comment by braver** [comment]
Ideally an error message contains a hint of how to solve it. So, if we’re sure globally installing something is the right solution we can suggest that.

Printing PATH feels like debugging to me. I would prefer something like “if this problem persists, please refer to our troubleshooting guide: (url)” and the readme of this plugin. 

Perhaps we should do that anyway, so the plugin readme can tell you what the right approach to installation etc is.

**Comment by ew4321** [comment]
I hope the changes I made are satisfactory.

**Comment by braver** [comment]
closed via #1502
